src/GuestConfiguration/GuestConfiguration/help/Get-AzVMGuestPolicyStatusHistory.md

```diff
@@ -66,26 +60,33 @@ ShowOnlyChange switch shows only historical status changes.
 Skips statuses that have not changed between two compliance checks.
 
 ### Example 4
-```
+```powershell
 PS C:\> Get-AzVMGuestPolicyStatusHistory -ResourceGroupName "MyResourceGroupName" -VMName "MyVMName" -InitiativeId "/providers/Microsoft.Authorization/policySetDefinitions/3fa7cbf5-c0a4-4a59-85a5-cca4d996d5af"
 ```
 
 Gets compliance status history by initiative Id.
 
 ### Example 5
-```
+```powershell
 PS C:\> Get-AzVMGuestPolicyStatusHistory -ResourceGroupName "MyResourceGroupName" -VMName "MyVMName" -InitiativeName "b5a822e0-ba98-4e54-9278-5d9833aa9b17"
 ```
 
 Gets compliance status history by initiative name.
 
 ### Example 6
-```
+```powershell
 PS C:\> Get-AzVMGuestPolicyStatusHistory -ResourceGroupName "MyResourceGroupName" -VMName "MyVMName"
 ```
-
 Gets compliance status history for all guest configuration policies assigned to the VM.
 
+### Example 7
+```powershell
+PS C:\> Get-AzVMGuestPolicyStatus -ReportId "/subscriptions/4e6c6ed2-0bf6-41d7-9d21-a452c2cc7920/resourceGroups/MyResourceGroupName/providers/Microsoft.Compute/virtualMachines/MyVMName/providers/Microsoft.GuestConfiguration/guestConfigurationAssignments/MaximumPasswordAge/reports/c271f845-2c0a-4456-a441-e48fc332d0ac"
```

Comment:
This isn't an example for this cmdlet - am I misunderstanding?

Reply:
hi, The idea is to get reportId from Get-AzVMGuestPolicyStatusHistory and use it in Get-AzVMGuestPolicyStatus. I put it here by design. I have changed the example though

Reply:
updated the example